x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -30,11 +30,7 @@ const PopoverContainer = styled.div`
   left: -40px;
   position: relative;
   top: 10px;
-
-  .euiPopover {
```

Comment:
`position: fixed` seemed to be what broke this. not sure what the `z-index` was for or if necessary @andrew-goldstein ?